cluster-operator/src/main/java/io/strimzi/operator/cluster/Main.java

```diff
@@ -109,8 +110,8 @@ static CompositeFuture run(Vertx vertx, KubernetesClient client, PlatformFeature
 
         List<Future> futures = new ArrayList<>();
         for (String namespace : config.getNamespaces()) {
-            Future<String> fut = Future.future();
-            futures.add(fut);
+            Promise<String> fut = Promise.promise();
```

Comment:
A stupid question. But should we try to rename also the env vars in these cases? Like in this example we have here now a `Promise` called `fut`. Few lines below is `Promise` called `returnFuture`. I know this will be hard, but if we don't do it this could be quite confusing.

Reply:
+1 to renaming the variable names.

Reply:
😬